module.json

```diff
@@ -15,12 +15,12 @@
   ],
 	"compatibility": {
 		"minimum": "10",
-		"verified": "10",
-		"maximum": "10"
+		"verified": "11",
+		"maximum": "11"
```

Comment:
Let's remove the `maximum` restriction, I don't think it's necessary in this case:
```suggestion
		"verified": "11"
```